.github/workflows/raise-pr-for-constants.yml

```diff
@@ -0,0 +1,81 @@
+name: Raise PR for Destination Constants
+
+on:
+  push:
+    branches:
+      - develop
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  BRANCH_NAME: chore/gh-update-destinations-ts
+  SRC_FILE: generated/Destinations.ts
+  DEST_PATH: packages/analytics-js-common/src/constants/integrations/Destinations.ts
+
+jobs:
+  check_changes:
+    runs-on: ubuntu-latest
+    outputs:
+      pr_required: ${{ steps.compare_files.outputs.pr_required }}
+    steps:
+      - name: Checkout Source Repo
+        uses: actions/checkout@v4
+        with:
+          ref: develop
+
+      - name: Checkout Destination Repo
+        run: |
+          DEST_REPO="rudderlabs/rudder-sdk-js"
+          git clone https://github.com/${DEST_REPO}.git dest-repo || exit 1
+          cd dest-repo
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "noreply@github.com"
+          git config --global url."https://${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
+          git checkout -b ${{ env.BRANCH_NAME }}
```

Comment:
Let's include a unique identifier like a timestamp in the branch name. If two successive hot fix releases are made, this workflow will fail if the previous PRs are not merged.

Reply:
Updated as per our discussion